script.js

```diff
@@ -0,0 +1,69 @@
+const numbers = document.querySelectorAll('.button');
+const operations = document.querySelectorAll('.button_operation');
+const decimal = document.getElementById('dot');
+const clearButtons = document.querySelectorAll('.button_clear');
+const display = document.getElementById('display');
+let memoryCurrentNumber = 0;
+let memoryNewNumber = false;
+let memoryPendingOperation = '';
+
+numbers.forEach((e, i) => numbers[i].addEventListener('click', (e) => operandEnter(e.target.textContent)));
+
+let operandEnter = (number) => {
+  if (memoryNewNumber) {
+    display.value = number;
+    memoryNewNumber = false;
+  } else display.value === '0' ? display.value = number : display.value += number;
+  display.value = display.value.substring(0,10);
+};
+
+operations.forEach((e, i) => operations[i].addEventListener('click', (e) => operatorEnter(e.target.textContent)));
+
+let operatorEnter = (op) => {
+  let localOperationMemory = display.value;
+  const maxDigits = 10;
+  const limitDigits = 6;
```

Comment:
Each time `operatorEnter` function will be called, you will create these constants. But they are actually constant. So, replace their declaration outside the function.

Reply:
DONE!